src/Elasticsearch.Net/ElasticLowLevelClient.NoNamespace.cs

```diff
@@ -72,6 +74,12 @@ public LowLevelCrossClusterReplicationNamespace CrossClusterReplication
 			private set;
 		}
 
+		public LowLevelDataFrameNamespace DataFrame
```

Comment:
Is there some overlap in data_frames and transform APIs?

Reply:
Discussed, decided to merge as is and tackle discuss this in the wider clients setting